jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java

```diff
@@ -74,16 +75,16 @@ public String getBaggageItem(String key) {
     return this.baggage;
   }
 
-  public long getTraceID() {
-    return traceID;
+  public long getTraceId() {
```

Comment:
Those that had `s/ID/Id/` change. I think SpanContext was the only one important enough to keep backwards compatible.